src/Core/Configure.php

```diff
@@ -138,6 +138,31 @@ public static function check($var)
     }
 
     /**
+     * Used to get information stored in Configure. It's not
+     * possible to store `null` values in Configure.
+     *
+     * Acts as a wrapper around Configure::read() and Configure::check().
+     * The configure value fetched via this method is expected to exist. 
+     * In case it does not an exception will be thrown.
+     *
+     * Usage:
+     * ```
+     * Configure::get('Name'); will return all values for Name
+     * Configure::get('Name.key'); will return only the value of Configure::Name[key]
+     * ```
+     *
+     * @param string $var Variable to obtain. Use '.' to access array elements.
+     * @return mixed Value stored in configure.
+     * @throws \Cake\Core\Exception\Exception if the requested configuration is not set.
+     */
+    public static function get($var) {
+        if (static::check($var) === false) {
+            throw new Exception(sprintf('Expected "%s" configuration.', $var));
```

Comment:
The reason I opted for Exception over RuntimeException is mimicking how things in Configure class are already done: http://api.cakephp.org/3.1/source-class-Cake.Core.Configure.html#307
I'll change it to RuntimeException because that's what it is, IMHO, right?

The Exception in `dump()` though seems to be Runtime specific as well?